mesheryctl/internal/cli/root/app/view.go

```diff
@@ -0,0 +1,129 @@
+package app
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+
+	"github.com/ghodss/yaml"
+	"github.com/layer5io/meshery/mesheryctl/internal/cli/root/config"
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var (
+	viewAllFlag   bool
+	outFormatFlag string
+)
+
+var viewCmd = &cobra.Command{
+	Use:   "view <application name>",
+	Short: "Display application(s)",
+	Long:  `Displays the contents of a specific application based on name or id`,
+	Args:  cobra.MaximumNArgs(1),
```

Comment:
I think application name can have multiple words it would be best to grab all the args and merge them to form a name. Just like applyCmd.

Reply:
using applicationID if isID .